### DISCUSSION TITLE
Latency

### DISCUSSION THREAD

**Discussion by PerBacks** [SauceLLC/sauce4zwift#87] [General]
In window "nearby riders" one can select latency. I still don't understand how it is possible that some riders occasionally get negative latency. What does that mean/indicate?

**Comment by mayfield** [comment]
That will change in the next release but it was because the latency was offset by the local latency between Sauce and the Game client instead of Sauce and Zwift servers.  In v1.0.0 latency will always be positive offsets from Sauce to Zwift.
